src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
 // write code here
+
+const listaUl = document.querySelector('ul');
+const listaLi = listaUl.querySelectorAll('li');
```

Comment:
Ensure that the 'ul' element exists before attempting to query for 'li' elements. This can prevent potential errors if the 'ul' element is not found on the page. You can add a conditional check before querying for 'li' elements.

Reply:
fix that issue